client/my-sites/google-my-business/connect-button/index.js

```diff
@@ -0,0 +1,135 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { last, isEqual } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { deleteStoredKeyringConnection } from 'state/sharing/keyring/actions';
+import { SharingService, connectFor } from 'my-sites/sharing/connections/service';
+import { getKeyringServiceByName } from 'state/sharing/services/selectors';
+import QueryKeyringServices from 'components/data/query-keyring-services';
+
+class GoogleMyBusinessConnectButton extends SharingService {
+	static propTypes = {
+		...SharingService.propTypes,
+		deleteStoredKeyringConnection: PropTypes.func,
+		onClick: PropTypes.func,
+		onConnect: PropTypes.func,
+	};
+
+	static defaultProps = {
+		...SharingService.defaultProps,
+		deleteStoredKeyringConnection: () => {},
```

Comment:
Super minor nitpick:
Maybe lodash's `noop` is more clear?
Also it's a single function.